clipping_manager/models/clipping.py

```diff
@@ -79,6 +79,16 @@ def save(self, *args, **kwargs):
         self._generate_content_hash()
         super(Clipping, self).save(*args, **kwargs)
 
+    def clear(self):
+        # Clear eveything besides content_hash
+        # and update deleted status
+        self.content = ""
+        self.book = None
+        self.author_name = ""
+        self.url = ""
+        self.deleted = True
+        super().save()
```

Comment:
Out of curiosity - why do we need the `super().save()` (instead of `save()` here?) 🤔

Reply:
That's because you've overwritten the save method, which now calls `_generate_content_hash` (redundant for empty content)

Reply:
Ah got it. We could either only run the hash generation when the object is created / saved for the first time, or could also add a param to to `save` method to allow skipping skip it.